lib/lotus/commands/generate.rb

```diff
@@ -34,6 +34,8 @@ def initialize(type, app_name, name, env, cli)
 
         @source   = Pathname.new(::File.dirname(__FILE__) + "/../generators/#{ @type }/").realpath
         @target   = Pathname.pwd.realpath
+
+        @env = env
```

Comment:
It will use to  call when will generate model throught entity and repository.
Here is it. https://github.com/lotus/lotus/pull/199/files#diff-c559e287fac190429de38e35866c4070R14